modules/end-user-guide/pages/microsoft-visual-studio-code-open-source-ide.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: CONCEPT
+:description: Microsoft Visual Studio Code - Open Source IDE
+:keywords: user-guide, selecting, IDE, select, selecting, editor, ide, vs-code, visual-studio, visual-studio-code, jetbrains, intellij, pycharm, theia, che-theia
+:navtitle: Microsoft Visual Studio Code - Open Source
+:page-aliases:
+
+[id="microsoft-visual-studio-code-open-source-ide"]
+= Microsoft Visual Studio Code - Open Source IDE
+
+The {prod-short} build of link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source] is the default IDE of a new workspace.
+
+You can have extensions automatically installed in this IDE for workspaces, as explained in the next section, from the xref:administration-guide:extensions-for-microsoft-visual-studio-code-open-source.adoc[Open VSX registry].
+
+TIP: You can use the link:https://code.visualstudio.com/api/ux-guidelines/command-palette[Command Palette] to find and run the commands specified in `devfile.yaml`.
+
+[TIP]
+====
+
+You can use *{prod2}* commands by clicking *{prod2}* in the link:https://code.visualstudio.com/api/ux-guidelines/status-bar[Status Bar] or finding them through the link:https://code.visualstudio.com/api/ux-guidelines/command-palette[Command Palette]:
+
+* *{prod2}: Open Dashboard*
+* *{prod2}: Stop Workspace*
+* *{prod2}: Restart Workspace*
```

Comment:
`Restart Workspace from Local Devfile` action was added to Eclipse Che commands within the PR recently: https://github.com/che-incubator/che-code/pull/185.

I think it makes sense to add the `Restart Workspace from Local Devfile` to this section.
Is it possible to do it within the current PR?
or
Should I create a separate PR for that?